quarkus/pom.xml

```diff
@@ -31,27 +31,8 @@
     <packaging>pom</packaging>
 
     <properties>
-        <!--
-            Do not change the properties bellow.
-            Their names should match the name used by Quarkus BOM and the versions are automatically set when running the 'set-quarkus-version.sh' script.
-        -->
-        <resteasy.version>4.7.7.Final</resteasy.version>
-        <hibernate-orm.version>5.6.14.Final</hibernate-orm.version>
-        <mysql-jdbc.version>8.0.30</mysql-jdbc.version>
-        <postgresql-jdbc.version>42.5.0</postgresql-jdbc.version>
         <wildfly-common.version>1.5.4.Final-format-001</wildfly-common.version>
-        <wildfly-elytron.version>1.20.1.Final</wildfly-elytron.version>
         <microprofile-metrics-api.version>3.0.1</microprofile-metrics-api.version>
-
-        <!--
-            We need to override certain dependencies from the parent POM.
-            TODO: remove these properties after changing the parent POM to use the same property names used by Quarkus BOM.
-        -->
-        <hibernate.core.version>${hibernate-orm.version}</hibernate.core.version>
-        <mysql.driver.version>${mysql-jdbc.version}</mysql.driver.version>
-        <postgresql.version>${postgresql-jdbc.version}</postgresql.version>
-        <wildfly.common.version>${wildfly-common.version}</wildfly.common.version>
-
         <!--
             Java EE dependencies. Not available from JDK 11+.
             The dependencies and their versions are the same used by Wildfly distribution.
```

Comment:
What about these? I assume that we can remove them too at this point. (I wasn't completely sure though, therefore I had left them out)

Reply:
I think we can remove them.

Reply:
Thanks, I addressed it in a34d0f7.